pom.xml

```diff
@@ -59,8 +59,10 @@
         <commons-io.version>2.4</commons-io.version>
         <mockito.version>1.10.19</mockito.version>
         <slf4j.version>1.7.5</slf4j.version>
-        <licenses.version>3.3.0-SNAPSHOT</licenses.version>
+	<licenses.name>Apache License 2.0</licenses.name>
```

Comment:
locally I used one tab instead of two which looks a lot different than on here. Don't really have an excuse but the latest commit should fix this